src/chocolatey/infrastructure.app/services/ISourceRunner.cs

```diff
@@ -25,122 +25,164 @@ namespace chocolatey.infrastructure.app.services
     using results;
 
     [MultiService]
-    public interface ISourceRunner
+    [Obsolete("This interface is deprecated and will be removed in v3.")]
+    public interface ISourceRunner : IBootstrappableSourceRunner, ICountSourceRunner, IListSourceRunner, IInstallSourceRunner, IUpgradeSourceRunner, IUninstallSourceRunner

```

Comment:
If this is being deprecated here, we should probably(?) make the NugetService / INugetService _also_ no longer use it? Currently I think it inherits off ISourceRunner directly; we should probably use the piece-wise interface for that as well.

Reply:
I seem to remember this being discussed, and the long and short of it was we decided to leave this in place for now.  There was quite a bit of discussion on this at the time, but it was also in between lots of other discussions, so I don't recall the exact details. We can discuss further once everyone is back from vacation, etc.

For now, watching the automated tests, to see about moving forward with taking some release candidates for a spin...